doc/files/source_stream-media.js.html

```diff
@@ -1257,6 +1297,11 @@ <h1 class="file-heading">File: source/stream-media.js</h1>
  *   For Chrome/Opera/IE/Safari/Bowser, the echo cancellation functionality may not work and may produce a terrible
  *   feedback. It is recommended to use headphones or other microphone devices rather than the device
  *   in-built microphones.&lt;/blockquote&gt; The flag to enable echo cancellation for audio track.
+ * @param {String|Array} [mediaSource=screen] The screensharing media source to select.
+ *   &lt;small&gt;Note that multiple sources (like providing an Array) is not supported by Firefox. The first source item
```

Comment:
Note that multiple sources are not supported by Firefox as of the time of this release. Firefox will use the first item specified in the Array in the event that multiple sources are defined.